http-app-dependencies/pom.xml

```diff
@@ -4,24 +4,43 @@
 	<modelVersion>4.0.0</modelVersion>
 	<groupId>org.springframework.cloud.stream.app</groupId>
 	<artifactId>http-app-dependencies</artifactId>
-	<version>2.0.2.BUILD-SNAPSHOT</version>
+	<version>2.0.3.BUILD-SNAPSHOT</version>
 	<packaging>pom</packaging>
 	<name>http-app-dependencies</name>
 	<description>Spring Cloud Stream HTTP App Dependencies</description>
 
 	<parent>
 		<artifactId>spring-cloud-dependencies-parent</artifactId>
 		<groupId>org.springframework.cloud</groupId>
-		<version>2.0.2.RELEASE</version>
+		<version>2.1.0.BUILD-SNAPSHOT</version>
 		<relativePath/>
 	</parent>
 
+	<properties>
+		<spring-cloud-function.version>2.0.0.BUILD-SNAPSHOT</spring-cloud-function.version>
+	</properties>
+
 	<dependencyManagement>
 		<dependencies>
 			<dependency>
 				<groupId>org.springframework.cloud.stream.app</groupId>
 				<artifactId>spring-cloud-starter-stream-source-http</artifactId>
-				<version>2.0.2.BUILD-SNAPSHOT</version>
+				<version>2.0.3.BUILD-SNAPSHOT</version>
+			</dependency>
+			<dependency>
+				<groupId>org.springframework.cloud.stream.app</groupId>
+				<artifactId>app-starters-function-common</artifactId>
+				<version>2.0.3.BUILD-SNAPSHOT</version>
```

Comment:
I don't think we need this dependency here, especially with the version.
See similar approach for all the `org.springframework.cloud.stream.app` deps in the SFTP, for example: https://github.com/spring-cloud-stream-app-starters/sftp/blob/master/spring-cloud-starter-stream-source-sftp/pom.xml#L29

You see? That comes without version and only in the target module, but definitely not root...